Crashes at initial state after activating junit4 plugin

Reinstalled the junit4 plugin to upgrade it and whenever I try to run repobee, with any command, I get this stack trace
```
[ERROR] Plugin 'junit4' does not allow 'reference_tests_dir' to be configured
exception was raised before pre-initialization was complete. This is usually due to incorrect settings. Try running the `verify-settings` command and see if the problem can be resolved. If all fails, please open an issue at https://github.com/repobee/repobee/issues/new and supply the stack trace below.
[ERROR] Critical exception
Traceback (most recent call last):
  File "/Users/glenn/.repobee/env/lib/python3.8/site-packages/_repobee/main.py", line 166, in _main
    parsed_args, api = _parse_args(
  File "/Users/glenn/.repobee/env/lib/python3.8/site-packages/_repobee/main.py", line 225, in _parse_args
    parsed_args, api = _repobee.cli.parsing.handle_args(
  File "/Users/glenn/.repobee/env/lib/python3.8/site-packages/_repobee/cli/parsing.py", line 56, in handle_args
    args, processing = _parse_args(sys_args, config_file)
  File "/Users/glenn/.repobee/env/lib/python3.8/site-packages/_repobee/cli/parsing.py", line 80, in _parse_args
    parser = cli.mainparser.create_parser(config_file)
  File "/Users/glenn/.repobee/env/lib/python3.8/site-packages/_repobee/cli/mainparser.py", line 131, in create_parser
    _add_subparsers(parser, config_file)
  File "/Users/glenn/.repobee/env/lib/python3.8/site-packages/_repobee/cli/mainparser.py", line 223, in _add_subparsers
    pluginparsers.add_plugin_parsers(
  File "/Users/glenn/.repobee/env/lib/python3.8/site-packages/_repobee/cli/pluginparsers.py", line 37, in add_plugin_parsers
    _attach_command(
  File "/Users/glenn/.repobee/env/lib/python3.8/site-packages/_repobee/cli/pluginparsers.py", line 69, in _attach_command
    cmd.attach_options(config=parsed_config, parser=ext_parser)
  File "/Users/glenn/.repobee/env/lib/python3.8/site-packages/repobee_plug/_pluginmeta.py", line 192, in _attach_options
    raise exceptions.PlugError(
repobee_plug.exceptions.PlugError: Plugin 'junit4' does not allow 'reference_tests_dir' to be configured
[ERROR] A plugin exited with an error
```
This was running just `repobee`

I assume it has to do with a `reference_tests_dir` property in my config but now I cannot even localise the config file, and I don't remember where it is hehe...

--- a/docs/faq.rst
+++ b/docs/faq.rst
@@ -82,8 +82,54 @@ otherwise changed after installing RepoBee. To fix this, remove the
 RepoBee crashes
 ===============
 
-First try to upgrade RepoBee to the latest version with ``repobee manage
-upgrade``. If that does not resolve the issue, try uninstalling plugins
-one-by-one with ``repobee --no-plugins plugin uninstall`` until RepoBee works
-again. If that has no effect either, then you should remove the ``~/.repobee``
-directory and execute the installer again (see :ref:`install`).
+Very often, RepoBee crashing is due to a faulty plugin, or bad configuration of
+a plugin. When this is the case, use the ``--no-plugins`` option for the
+preparser to disable plugins.
+
+.. note::
+
+    ``--no-plugins`` is a *preparser* option, meaning that it should be passed
+    directly after typing ``repobee``. For example, to run ``repobee plugin
+    uninstall`` without plugins, run like so:
+
+    .. code-block::
+        :caption: Uninstalling plugins with all installed plugins disabled
+
+        $ repobee --no-plugins plugin uninstall
+
+All of the troubleshooting steps in this section will use ``--no-plugins``.
+
+An option in the config file is causing problems
+------------------------------------------------
+
+If you have misconfigured a plugin, or have an outdated option, you may
+sometimes need to  manually remove the option from the config file.
+Run the following to find out where the config file is located.
+
+.. code-block::
+    :caption: Finding out where the config file is located
+
+    $ repobee --no-plugins config show
+
+The location of the config file is printed at the very top of the message.
+Edit it with any text editor and remove the option that causes issues.
+
+A plugin has broken down
+------------------------
+
+If a plugin isn't working (even after removing faulty options), try to upgrade
+both the plugin and RepoBee itself.
+
+.. code-block::
+    :caption: Upgrade RepoBee and install the plugin again
+
+    $ repobee --no-plugins manage upgrade
+    $ repobee --no-plugins plugin install
+
+Nothing works, it's still broken
+--------------------------------
+
+If none of that works, the simplest thing is to reinstall RepoBee. This does
+*not* affect your settings in the config file, it only affects installed
+plugins, and so is low-risk to do. To do it, just remove the directory at
+``~/.repobee`` and install RepoBee again, as described in :ref:`install`.

